orion-ui/tests/utilities/useToggle.ts

```diff
@@ -0,0 +1,55 @@
+import { Locator, Page } from '@playwright/test'
+import { PAGE } from './fixtures'
+
+export type UseToggle = {
+  toggle: Locator,
+  getState: () => Promise<boolean | undefined>,
+  setState: (state: boolean) => Promise<void>,
+}
+
+export function useToggle(page: Page | Locator = PAGE): UseToggle {
```

Comment:
This appears to act on all toggles within the page or locator which doesn't seem ideal. Could this work on a single toggle and fail if multiple toggles are found within a locator? I can't think of a use case where we would want to act on multiple toggles at the same time. And if we do, useToggle should probably be called multiple times to intentionally select the desired toggles.

Reply:
And by fail I mean just have it return the locates toggle(s) but any methods that do actions would fail because of playwrights strict check on actions being called on single locators

Reply:
yeah I like that. I was trying to follow what I thought was the existing expectation that these utilities work the same on a single element as they do a list

Reply:
No I don't think any of the utilities work on a list.